lib/types.ts

```diff
@@ -6,7 +6,7 @@ export interface SbSvelteComponentsMap {
 }
 
 export interface SbSvelteSDKOptions extends SbSDKOptions {
-  components?: SbSvelteComponentsMap;
+  components?: SbSvelteComponentsMap | CallableFunction;
```

Comment:
well done 👍

Reply:
Actually, instead of typing it as just a function, we could specify that is a function that returns a `SbSvelteComponentsMap`. Asking for @mmatheuspalma expertise here ;)